src/_includes/cloud/cloud-elasticsearch-client-compatibility.md

```diff
@@ -4,4 +4,4 @@ Elasticsearch service   |  Elasticsearch composer package | Status
 --------- | ------------- | -------------------------
 5.2.x | 5.x.x | default versions for {{ site.data.var.ee }} versions 2.2.x to 2.2.7 and 2.3.0
 6.8 | 6.7.x | recommended, default version for {{ site.data.var.ee }} versions 2.2.8 and 2.3.4
-7.5, 7.6, 7.7 | 7.x | recommended versions for {{ site.data.var.ee }} versions 2.3.5 and later
+7.7 | 7.x | recommended versions for {{ site.data.var.ee }} 2.3.5 and later versions, 2.4 and later versions
```

Comment:
Seems like a bit of inconsistency with the other two lines. 
Also, if you say 2.3.5 and later versions, then why list a later version?

```suggestion
7.7 | 7.x | recommended, default versions for {{ site.data.var.ee }} versions 2.3.5 and 2.4.x
```